### DISCUSSION TITLE
Combine different analytical solutions

### DISCUSSION THREAD

**Discussion by anubhavkamal** [MilesCranmer/PySR#546] [Q&A]
Hello!
I have a problem statement as follows -
There are multiple datasets formulated when we change input current in a circuit(constant current input). I applied PySR on each dataset to get multiple analytical solutions.
Is there a way to combine these solutions and get a solution as a function of current too?
Thanks

**Comment by anubhavkamal** [comment]
@MilesCranmer

**Comment by MilesCranmer** [comment]
Could you be a bit more specific about what you want to do? Do you want to basically compose the equations together?
